templates/identity/email-password/register-response.html.ejs

```diff
@@ -37,18 +37,18 @@
     <% } %>
 
     <% if (register) { %>
-      <h2>Your new login</h2>
+      <h2>Your new account</h2>
       <p>
-        You can <a href="./login">log in</a>
-        with your WebID <a href="<%= webId %>" class="link"><%= webId %></a>
-        on this server via your email address <em><%= email %></em>.
+        Via your email address <em><%= email %></em>,
+        this server lets you <a href="./login">log in</a> to Solid apps
```

Comment:
Should we have a note saying the login page on this link is actually not going to work if you browsed directly to the register page. This might be a common case since many people might end up on the registration page through the explanation in the root index.html.

Reply:
https://github.com/solid/community-server/issues/880